### PR TITLE
refactor(server): update camera description

### PR DESCRIPTION
**Pull request by mkumbobeaty**
# Overview
Update descriptions for Scene/Camera properties’s description
## What I've done
- change the description for:
  - initial camera
  - Enter the ground
## What I haven't done
   I didn't change the description for FOV because it already changed in [slider](https://github.com/reearth/reearth-visualizer/pull/1266) Pr:

## Summary by CodeRabbit

- **New Features**
	- Enhanced descriptions for camera settings and terrain properties in the visualizer, improving user guidance.
	- Updated labels for scene mode options to provide clearer choices for users.

- **Documentation**
	- Improved clarity and specificity of descriptions for various configuration properties, ensuring better understanding for users.

### CODE REVIEW

File: server/pkg/builtin/manifest_ja.yml [hunk 1]
```diff
@@ -130,10 +130,10 @@ extensions:
         fields:
           camera:
             title: カメラの初期位置
-            description: プロジェクトを開いたときに表示される最初の位置
+            description: プロジェクトを開くたびに同じ領域が表示されるように、初期カメラ位置を設定できます。
           allowEnterGround:
             title: 地下に入る
-            description: 有効にすると、カメラが地表の中に入るようになります。
+            description: このプロパティが true に設定されている場合、カメラは地形内を移動でき、地下エリアに入ることができます。
```
Comment (asahina820) [inline server/pkg/builtin/manifest_ja.yml:136]: "有効にすると、カメラは地形の中を移動でき、地下エリアに入ることができます。" is better

File: server/pkg/builtin/manifest_ja.yml [hunk 2]
```diff
@@ -130,10 +130,10 @@ extensions:
         fields:
           camera:
             title: カメラの初期位置
-            description: プロジェクトを開いたときに表示される最初の位置
+            description: プロジェクトを開くたびに同じ領域が表示されるように、初期カメラ位置を設定できます。
```
Comment (asahina820) [inline server/pkg/builtin/manifest_ja.yml:133]: 初期カメラ位置を設定すると、プロジェクトを開くたびに同じ場所が表示されるようになります。  is better

**Review by asahina820 — COMMENTED** [review]
I checked Japanese translation 👍

File: server/pkg/builtin/manifest_ja.yml (hunk 2 above)
Comment (mkumbobeaty) [reply server/pkg/builtin/manifest_ja.yml:133]: thank you

**Review by airslice — APPROVED** [review]

**Review by pyshx — APPROVED** [review]

**Pull request merged**